### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN052.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#102] [General]
# https://blog.3place.cn/3place/lmyn/LMYN052.html

https://blog.3place.cn/3place/lmyn/LMYN052.html

**Comment by picklecai** [comment]
有大力量的毗奢蜜多罗，
兴高采烈快乐得无以复加，
他循规蹈矩谦恭自持，

这个描述，很子张。

**Comment by picklecai** [comment]
为什么突然看到婆私吒了？婆私吒是十车王的国师，这里不是十车王的城。

**Comment by picklecai** [comment]
如意神牛呀！让一切的东西，
都雨般落下，为我的缘故。 （1.51.22）

鹿、猴子和牛。

**Comment by picklecai** [comment]
婆私吒的问候：
1. 有没有使用王权保护百姓，
2. 仆人是否服从，
3. 敌人是否顺服，
4. 军队、财产、亲友是否平安

1是国王的行动
234是行动的效果

作者认为：以上4条，是一个国王最重要的事。

**Comment by picklecai** [comment]
这两个虔诚的人物，
愉快地谈了很久；
他俩都万分快乐，
彼此相爱成朋友。 （1.51.11）

只聊天，不涉及实际利益，太容易“朋友”了。

**Comment by picklecai** [comment]
那位世尊婆私吒，
又对毗奢蜜多罗，
微笑着说了话： （1.51.12）

’英武有力的人！我很想
对你的军队尽上地主之谊；
他们真是数也数不清，
但愿你对我表示同意。 （1.51.13）

我准备的这一点小意思，
就请陛下立刻俯予接受。
国王呀！你是最尊贵客人，
我们必须尽力把你侍候。’ （1.51.14）

和孙悟空显示袈裟不同，婆私吒显示神牛，原是为了待客之道。

**Comment by picklecai** [comment]
那聪明的毗奢蜜多罗，
他这个国王连忙说道：
’不要再把恭敬的话来说。 （1.51.15）

这里有的是果子和根茎，
世尊呀！在这净修林里；
洗脚水到处都可以打到，
还可以看到世尊你自己。 （1.51.16）

伟大的智者呀！我已经
把你的款待享受了一个遍。
我就要离开了，祝你幸福！
请用友谊的眼光来相看。’ （1.51.17）

众友觉得已有的款待已经足够了，心意已经收到。
这个推辞也十分得体。
